Formula/l/llama.cpp.rb

```diff
@@ -0,0 +1,41 @@
+class LlamaCpp < Formula
+  desc "LLM inference in C/C++"
+  homepage "https://github.com/ggerganov/llama.cpp"
+  # pull from git tag to get submodules
+  url "https://github.com/ggerganov/llama.cpp.git",
+      tag:      "b2963",
+      revision: "95fb0aefab568348da159efdd370e064d1b35f97"
+  license "MIT"
+
+  livecheck do
+    throttle 10
+  end
```

Comment:
I don't think this is needed. We can introduce this later if it's necessary for maintainers

Reply:
Thanks for the suggestion!